v1/pipeline/steps/cd/parallel-node.yaml

```diff
@@ -0,0 +1,39 @@
+title: ParallelNode
+type: object
+required:
+  - spec
+  - type
+properties:
```

Comment:
@shalini-agr can you point the common paralle and group files so that @markoharness can link those, you can share the example where you are used